packages/arui-scripts/src/configs/jest/index.js

```diff
@@ -34,7 +34,7 @@ const defaultJestConfig = {
     ],
     globals: {
         'ts-jest': {
-            tsConfig: configs.tsconfig,
+            tsconfig: configs.tsconfig,
```

Comment:
ts-jest[config] (WARN) The option `tsConfig` is deprecated and will be removed in ts-jest 27, use `tsconfig` instead